diff --git a/src/sdk/installerbase.qrc b/src/sdk/installerbase.qrc
index 55715a1..b14ebdb 100644
--- a/src/sdk/installerbase.qrc
+++ b/src/sdk/installerbase.qrc
@@ -8,7 +8,7 @@
         <file alias="translations/zh_CN.qm">translations/zh_cn.qm</file>
         <file>translations/qt_fr.qm</file>
         <file>translations/qt_de.qm</file>
-        <file>translations/qt_ja.qm</file>
+        <file alias="translations/qt_ja.qm">translations/qtbase_ja.qm</file>
         <file>translations/qt_ru.qm</file>
         <file>translations/qt_zh_CN.qm</file>
     </qresource>
diff --git a/src/sdk/sdk.pro b/src/sdk/sdk.pro
index 51098e3..41110d9 100644
--- a/src/sdk/sdk.pro
+++ b/src/sdk/sdk.pro
@@ -19,7 +19,7 @@ isEqual(QT_MAJOR_VERSION, 5) {
 DESTDIR = $$IFW_APP_PATH
 
 exists($$LRELEASE) {
-    QT_LANGUAGES = qt_de qt_fr qt_ja qt_ru qt_zh_CN
+    QT_LANGUAGES = qt_de qt_fr qtbase_ja qt_ru qt_zh_CN
     IB_LANGUAGES = de_de en_us fr_fr ja_jp ru_ru zh_cn
     defineReplace(prependAll) {
         for(a,$$1):result += $$2$${a}$$3
